### ISSUE TITLE
Let GitHub works for us and create PR for changes which comes after running internal scripts

### ISSUE THREAD

**Issue by bmarty** [element-hq/element-android#3902]
### Your use case

During the release we run some scripts to update the codebase from external resources, but it has to be done manually.

The [two steps](https://github.com/vector-im/element-android/blob/main/.github/ISSUE_TEMPLATE/release.md#L21) (so far) are:
- Run `./tools/import_emojis.py` and commit the change if any.
- Run `./tools/import_sas_strings.py` and commit the change if any.

It should be better to set up a GitHub cron or something similar to let GitHub run the script and create PR in case of any changes.

Maybe one cron for each script, hence the changes are not frequent, so maybe only one generic cron can be enough, but commit will have to be done after each script.

### Have you considered any alternatives?

No

### Additional context

Ideally, if any PR exists and has been created by the cron, no new PR should be created with the same change. If there are new change, existing PR should be deleted and replaced by the new one. A bit similar to what dependabot is doing to upgrade the gradle dependencies.

Once this is done, we could remove the 2 steps from the release template.
